pychromecast/controllers/plex.py

```diff
@@ -355,9 +364,13 @@ def status(self):
         Returns:
             pychromecast.controllers.media.MediaStatus: Slightly modified status with patched
                                                         method for episode_title.
+                                                        Overwrite supported_media_commands 
+                                                        when set to MEDIA_BASIC
         """
         status = self._socket_client.media_controller.status
         status.episode_title = episode_title
+        if status.supported_media_commands == 12303: 
+            status.supported_media_commands = 12543
```

Comment:
Let's not use magic numbers, use the `CMD_SUPPORT_*`-constants defined in media.py instead.

The second row is doing:
```py
    status.supported_media_commands |= CMD_SUPPORT_SKIP_FORWARD | CMD_SUPPORT_SKIP_BACKWARD | CMD_SUPPORT_QUEUE_NEXT | CMD_SUPPORT_QUEUE_PREV
```

The first row seems very specific, is it correct, or should we always set the bits indicating support for previous and next?